pnpm-lock.yaml

```diff
@@ -0,0 +1,11282 @@
+lockfileVersion: '6.0'
```

Comment:
Let's remove this file. We'll use yarn

Reply:
Fixed with [a76f4af](https://github.com/Suvink/let-me-react/pull/15/commits/a76f4af6e4662e78f6b4c5ee4e0a47a630c237dd)